internal/monitoring/monitoringserver.go

```diff
@@ -0,0 +1,57 @@
+package monitoring
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+// Monitoring is an instance of STUNner monitoring
+type MonitoringServer struct {
+	httpServer *http.Server
+	Endpoint   string
+	Group      string
+}
+
+// NewMonitoring initiates the monitoring subsystem
+func NewMonitoringServer(endpoint string, group string) (*MonitoringServer, error) {
```

Comment:
can we rename this to MonitoringBackend or PromBackend or smilar? it's not a server: the server side is handled in the Admin object, what we define here is merely a HTTP handler for that server.

Reply:
Done in [72def2e](https://github.com/l7mp/stunner/pull/12/commits/72def2e9cfdff6b8efd79d11b0205152b84dcd53).